Woof/Woof/NetworkLayer/Endpoint/Encodable+asDictionary.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+extension Encodable {
+    /// Converts the object to a dictionary representation.
```

Comment:
I suggest clarifying the method.
```suggestion
    /// Converts the object to a dictionary representation, where keys
    /// are the string representation of the stored properties' names and
     /// values are the values of those properties.
    ///
    /// During converting the camel case notation of properties' names replace with the snake case notation.
```

Reply:
Agreed, update the documentation.